services/database/programcardinfo.ts

```diff
@@ -0,0 +1,168 @@
+import prisma from "@database";
+import { ProgramCardInfo } from "models/Program";
+import { ClassCardInfo } from "models/Class";
+import { locale } from "@prisma/client";
+
+/**
+ * @param {string} id - programId
+ */
+async function getClassInfoWithProgramId(id: string): Promise<ClassCardInfo[]> {
```

Comment:
Related to comment above, let's just have a generic getClassesByProgramId query that's used in `api/class`.